api/TimeAddressableMediaStore.yaml

```diff
@@ -117,27 +117,339 @@ paths:
           description: Success. The service info has been updated.
         "400":
           description: Bad request. Invalid service JSON.
+
+
   /sources:
     head:
-      description: Return sources path headers
+      summary: List Sources
+      description: Return Sources path headers
       operationId: HEAD_sources
       tags:
         - Sources
       responses:
         "200":
           $ref: '#/components/responses/trait_resource_listing_head_200'
     get:
-      description: List the sources registered in the store.
+      summary: List Sources
+      description: List the Sources registered in the store and their details.
       operationId: GET_sources
       tags:
         - Sources
+      parameters:
+        - name: label
+          in: query
+          description: Filter on Sources that have the given label.
+          schema:
+            type: string
+        - name: tag.{name}
+          in: query
+          description: |
+            Filter on Sources that have a tag named {name} and with the given value.
+            The {name} could contain escaped characters to allow it to be used in a
+            URL.
+          schema:
+            type: string
+        - name: tag_exists.{name}
+          in: query
+          description: |
+            Filter on Sources that have a tag named {name} regardless of value. The
+            {name} could contain escaped characters to allow it to be used in a
+            URL. If set to true then the presence of the tag is filtered for. If set
+            to false then its absence is. If left out then no filtering on tag presence
+            is performed.
+          schema:
+            type: boolean
+        - name: is_root_node
+          in: query
+          description: |
+            Filter on Sources that have no parent Sources (specifically, no Collects relationship)
+            and form the root node of part of the hierarchy.
+          schema:
+            type: boolean
       responses:
         "200":
           description: ""
```

Comment:
Is the description here and on line 217 left blank intentionally?

Reply:
`description` is a mandatory field, but I'm unsure whether writing something like "List of Sources known to the API and their details" adds anything beyond what's already written in the route description? What do you think?

Reply:
I suppose you're right, it's a bit redundant - it just threw me off a bit!